Makefile

```diff
@@ -67,39 +88,35 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 		pip-compile --upgrade -o requirements/test-$$platform.txt requirements/base.in requirements/test-$$platform.in requirements/test.in ; \
 	done
 
-pull_translations: ## pull translations from Transifex
-	tx pull -a
-
-push_translations: ## push source translation files (.po) from Transifex
-	tx push -s
-
-quality: ## check coding style with pycodestyle and pylint
-	tox -e quality
+requirements.js: ## install JS requirements for local development
+	npm install
 
-jasmine: ## run Javascript tests
-	tox -e jasmine
+requirements: requirements.js ## install development environment requirements
+	pip install -qr requirements/dev.txt --exists-action w
+	pip-sync requirements/base.txt requirements/dev.txt requirements/private.* requirements/test.txt
 
 jshint: ## run Javascript linting
-	@[ -x ./node_modules/jshint/bin/jshint ] || npm install jshint
+	@[ -x ./node_modules/jshint/bin/jshint ] || npm install jshint --save-dev
 	./node_modules/jshint/bin/jshint enterprise
 	./node_modules/jshint/bin/jshint spec
 
-requirements: ## install development environment requirements
-	pip install -qr requirements/dev.txt --exists-action w
-	pip-sync requirements/base.txt requirements/dev.txt requirements/private.* requirements/test.txt
-
 test: clean ## run tests in the current virtualenv
 	pip install -qr requirements/test.txt --exists-action w
 	py.test
 
 diff_cover: test
 	diff-cover coverage.xml
 
-test-all: clean jshint quality jasmine
-	## run tests on every supported Python/Django combination
+test-all: clean jshint ## run tests on every supported Python/Django combination
 	tox
+	tox -e quality
+	tox -e jasmine
 
-validate: quality test ## run tests and quality checks
+validate: test ## run tests and quality checks
+	tox -e quality
 
-isort: ## Call isort on packages/files that are checked in quality tests
-	isort --recursive tests test_utils enterprise consent manage.py setup.py test_settings.py
+isort: ## call isort on packages/files that are checked in quality tests
+	isort --recursive tests test_utils enterprise consent manage.py setup.py
+
+.PHONY: clean compile_translations coverage docs dummy_translations extract_translations \
+	fake_translations help pull_translations push_translations requirements test test-all upgrade validate isort
```

Comment:
The static make commands should be in here too, correct?

Reply:
Yep, good catch :+1: